apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -18,6 +18,7 @@ package v1alpha1
 
 import (
```

Comment:
Yes, maybe it could reduce amount of dependencies required, but looks like not so significant. Please see below. Thanks
$ cd integreatly-operator
$ find . -name "*.go" |xargs grep -l github.com/3scale/3scale-operator/apis/apps |wc -l
19
$ find . -name "*.go" |xargs grep -l github.com/3scale/3scale-operator/apis/capabilities |wc -l
5